apps/api_web/test/api_web/plugs/rate_limiter_test.exs

```diff
@@ -58,4 +58,20 @@ defmodule ApiWeb.Plugs.RateLimiterTest do
       assert json_response(conn, :forbidden)["errors"]
     end
   end
+
+  describe "requests" do
+    setup %{conn: conn} do
+      ApiWeb.RateLimiter.force_clear()
+      {:ok, conn: conn}
+    end
+
+    test "have rate limiting headers in response", %{conn: conn} do
+      conn = get(conn, @url)
+      resp_headers = conn.resp_headers
+
+      assert {"x-ratelimit-limit", _} = List.keyfind(resp_headers, "x-ratelimit-limit", 0)
```

Comment:
You can use [`Plug.Conn.get_resp_header/2`](https://hexdocs.pm/plug/Plug.Conn.html#get_resp_header/2) for this:
```elixir
assert [_] = get_resp_header(conn, "x-rate-limit-limit")
...
```